code/api/src/modules/shipments/mutations.js

```diff
@@ -0,0 +1,34 @@
+// Imports
+import { GraphQLInt, GraphQLString} from 'graphql'
+
+// App Imports
+import ShipmentType from './types'
+import { create, update } from './resolvers'
+
+// Shipment create
+export const shipmentCreate = {
+  type: ShipmentType,
+  args: {
+    subscriptionId: {
+      name: 'subscriptionId',
+      type: GraphQLInt
+    }
+  },
+  resolve: create
+}
+
+// Delivery Date Update
+export const deliveryUpdate = {
+  type: ShipmentType,
+  args: {
+    id: {
+      name: 'id',
+      type: GraphQLInt
+    },
+    deliveryDate: {
```

Comment:
this was a good call to only allow for the date to be edited

Reply:
I agree with that! No need to alter any of the other data coming through.